api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/EriDSDDataset.java

```diff
@@ -533,7 +511,8 @@ public DataSetDefinition constructEriDSDDataset() {
     ColumnParameters twoTo4 = new ColumnParameters("twoTo4", "2-4", "age=2-4", "01");
     ColumnParameters fiveTo9 = new ColumnParameters("fiveTo9", "5-9", "age=5-9", "02");
     ColumnParameters tenTo14 = new ColumnParameters("tenTo14", "10-14", "age=10-14", "03");
+    ColumnParameters lesThan2 = new ColumnParameters("lesThan2", "<2", "age=<2", "04");
```

Comment:
in the template I cant see dis-aggregation for <2 in Eligible (Stable)

Reply:
We are reusing this list but not mapping <2 yrs patients on eligible. Eligible patents must be >2years

Reply:
ok. noted